Testing string equality with 'test' is done with operator '=', not '=='

--- configure~	2011-07-07 21:25:39.000000000 +0200
+++ configure	2011-07-07 21:27:35.000000000 +0200
@@ -5344,7 +5344,7 @@
 done
 test -n "$PCRE_CONFIG" || PCRE_CONFIG="no"
 
-    if test "$PCRE_CONFIG" == "no"; then :
+    if test "$PCRE_CONFIG" = "no"; then :
 
       :
       { { $as_echo "$as_me:${as_lineno-$LINENO}: error: in \`$ac_pwd':" >&5
@@ -5366,7 +5366,7 @@
 
 else
 
-            if test x"" == x; then :
+            if test x"" = x; then :
 
 	PCRE_CFLAGS="`$PCRE_CONFIG $pcre_config_args --cflags`"
 
@@ -5376,7 +5376,7 @@
 
 fi
 
-            if test x"" == x; then :
+            if test x"" = x; then :
 
 	PCRE_LIBS="`$PCRE_CONFIG $pcre_config_args --libs`"
 
@@ -9205,19 +9205,19 @@
 	isolate_b_regex='\([0-9]\+\).*'
 	for ver_part in $nodots_a ; do
 		b_ver_part=`echo "$ver_b" | sed -e 's/'"$isolate_b_regex"'/\1/'`
-		if test \( "$ver_part" -lt "$b_ver_part" \) -a \( "x$condition" == "xequal" \) ; then
+		if test \( "$ver_part" -lt "$b_ver_part" \) -a \( "x$condition" = "xequal" \) ; then
 			condition=less
-		elif test \( "$ver_part" -gt "$b_ver_part" \) -a \( "x$condition" == "xequal" \) ; then
+		elif test \( "$ver_part" -gt "$b_ver_part" \) -a \( "x$condition" = "xequal" \) ; then
 			condition=greater
 		fi
 		isolate_b_regex='[0-9]\+\.'"$isolate_b_regex"
 	done
 
-	if test "x$condition" == "xequal" ; then
+	if test "x$condition" = "xequal" ; then
 		:
-	elif test "x$condition" == "xless" ; then
+	elif test "x$condition" = "xless" ; then
 		:
-	elif test "x$condition" == "xgreater" ; then
+	elif test "x$condition" = "xgreater" ; then
 		OCAMLLOC=_loc
 	fi
 
